src/Tickets/Seating/Orders/Seats_Report.php

```diff
@@ -127,11 +134,12 @@ public function setup_template_vars(): array {
 		$ephemeral_token     = tribe( Service::class )->get_ephemeral_token( 6 * HOUR_IN_SECONDS, 'admin' );
 		$token               = is_string( $ephemeral_token ) ? $ephemeral_token : '';
 		$this->template_vars = [
-			'post'       => $post,
-			'post_id'    => $post_id,
-			'iframe_url' => tribe( Service::class )->get_seat_report_url( $token, $post_id ),
-			'token'      => $token,
-			'error'      => $ephemeral_token instanceof WP_Error ? $ephemeral_token->get_error_message() : '',
+			'post'          => $post,
+			'post_id'       => $post_id,
+			'iframe_url'    => tribe( Service::class )->get_seat_report_url( $token, $post_id ),
+			'token'         => $token,
+			'error'         => $ephemeral_token instanceof WP_Error ? $ephemeral_token->get_error_message() : '',
+			'should_upsell' => $this->should_show_upsell(),
```

Comment:
I would keep the same var name:

```suggestion
			'should_show_upsell' => $this->should_show_upsell(),
```